### PR TITLE
Format all files with Black

### PR DESCRIPTION
**Pull request by sgoldenCS**
Ran black using `hatch run style:format` which formatted 21 files in our repository. This is a big change that might require some checking to ensure everything is good. Our numpy_reader unit test still runs without errors, and the hydra_driver seems to work fine as well.

### CODE REVIEW

File: Hall_B/AIDAPT/aidapt_toolkit/data_parsers/aidapt_csv_reader_v0.py [hunk 1]
```diff
@@ -94,50 +94,49 @@ def save(self, path: str):
...
-            'Calling load()...')
+            "load_config() is currently unimplemented. " "Calling load()..."
+        )
         return self.load(path)
 
     def save_config(self, path: str):
         aidapt_numpy_reader_log.debug(
-            'save_config() is currently unimplemented.'
-            ' Calling save()...?')
+            "save_config() is currently unimplemented." " Calling save()...?"
```
Comment (sgoldenCS) [inline Hall_B/AIDAPT/aidapt_toolkit/data_parsers/aidapt_csv_reader_v0.py:140]: We should remove the internal double quotes here. Functionally it is the same, but it looks odd. I think this happened in multiple files.

File: Hall_B/AIDAPT/aidapt_toolkit/drivers/basic_inner_gan_driver.py [hunk 2]
```diff
@@ -40,41 +54,81 @@ def run(config):
...
-    lab2inv_prep.save(os.path.join(config['driver']['save_path'], 'lab2inv_prep'))
-    d_scaler.save(os.path.join(config['driver']['save_path'], 'd_scaler'))
-    v_scaler.save(os.path.join(config['driver']['save_path'], 'v_scaler'))
-    model.save(os.path.join(config['driver']['save_path'], 'cgan_model'))
-
-if __name__ == '__main__':
+    fig, ax = plt.subplots(1, 1)
+    p_rec_gan = np.sqrt(
+        (
+            np.power(d_results[:, 0], 2)
```
Comment (sgoldenCS) [inline Hall_B/AIDAPT/aidapt_toolkit/drivers/basic_inner_gan_driver.py:89]: I think `np.power(x,2)` could be replaced with `x**2`

File: Hall_B/AIDAPT/aidapt_toolkit/utils/config_utils.py [hunk 3]
```diff
@@ -5,5 +5,7 @@ def verify_config(config, required_keys):
             missing_keys.append(key)
 
     if missing_keys:
-        error_msg = f'Config is missing the following required keys: {", ".join(missing_keys)}'
-        raise KeyError(error_msg)
\ No newline at end of file
+        error_msg = (
+            f'Config is missing the following required keys: {", ".join(missing_keys)}'
```
Comment (sgoldenCS) [inline Hall_B/AIDAPT/aidapt_toolkit/utils/config_utils.py:9]: Maybe there's a better way to do this to avoid 3 lines for a relatively short string...

**Comment by sgoldenCS** [comment]
For clarification: `black`[(link)](https://github.com/psf/black) is a command-line tool that can be installed with `pip install black` that automatically formats python code based on a standard set of rules (following PEP8) that govern
- Whitespace
  - No trailing whitespace
  - Rules about spaces around operators and `=`
- Line length
  - Limited by default to 88 characters
- String Format
  - Strings are delimited by double quotes `"` not single `'` 
- Etc. 

This pull request does not include any manual changes to `black`'s configuration or to the code itself. All changes were done using `hatch run style:format` (or equivalently `black .` in the repo's main directory).

**Comment by dlersch** [comment]
This looks good. thank you for the clarification.

**Pull request merged**